homeassistant/components/glances/sensor.py

```diff
@@ -41,7 +41,7 @@
     'process_thread': ['Thread', 'Count', 'mdi:memory'],
     'process_sleeping': ['Sleeping', 'Count', 'mdi:memory'],
     'cpu_use_percent': ['CPU used', '%', 'mdi:memory'],
-    'cpu_temp': ['CPU Temp', TEMP_CELSIUS, 'mdi:thermometer'],
+    'cpu_temp': ['CPU Temp', '°C', 'mdi:thermometer'],
```

Comment:
I don't see how this change matters? This is the exact same value as `TEMP_CELSIUS`

Reply:
https://github.com/home-assistant/home-assistant/pull/6433#discussion_r106589509

Unless this comment is no longer true, having `preferred_units` set to `imperial` means that it will automatically convert the CPU temperatures to Fahrenheit. Pulling the CPU temperatures directly from the API displays them in Celsius (you can override this with a glances command option, but by default its Celsius)

Reply:
`TEMP_CELSIUS` is a constant of `°C`. That mean, your code don't anything.